alpha-build-machinery/make/targets/golang/verify-update.mk

```diff
@@ -26,6 +26,16 @@ verify-golint:
 	$(GOLINT) $(GO_PACKAGES)
 .PHONY: verify-govet
 
+verify-sec:
+ifeq (, $(shell which $(GOSEC) 2>/dev/null))
+	go get -u github.com/securego/gosec/cmd/gosec
```

Comment:
messing with GOPATH is not desired. I could go for a message and where to get it

Reply:
A message does not work. This is the same problem we had with crd-schema-gen. We have to put versions into our makefiles, not anywhere else (I could live with a variable to override it from an operator, but that won't make a difference to this argument).